### ISSUE TITLE
Extend narrative parsing to include translators

### ISSUE THREAD

**Issue by jameshadfield** [nextstrain/auspice#933]
Currently the YAML frontmatter at the start of each narrative markdown file is parsed [by this code](https://github.com/nextstrain/auspice/blob/master/cli/server/parseNarrative.js#L23) into HTML to be rendered by the client as the fist page of the narrative. E.g. [this YAML](https://github.com/nextstrain/narratives/blob/master/ncov_sit-rep_2020-03-05.md) tuns into [this narrative page](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-05).

Currently we display YAML fields `title`, `authors`, `authorLinks`, `affiliations`, `date` and `updated`.

Given the fantastic work by the community to translate the nCoV narratives, we should extend this to also parse: `translators` -> array of strings, each containing an optional URL or email. E.g. the current Spanish translation would include:

```yaml
translators:
  - Julián Villabona-Arenas <https://twitter.com/Chjulian>
```

And auspice should display this as a line below the author affiliations

**Comment by ncallaway** [comment]
If no one else has started working on this, I'll take a look at this tonight.

**Comment by jameshadfield** [comment]
Closed by #937
